### ISSUE TITLE
Accessibility Feedback for Midpoint Review [VSA Caregivers, Sign as a Representative)

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#23222]
## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh and Trevor

## Must
- [Dev intent] Ensure `legend`s are properly used with `fieldset`s 

## Should
- Use button styling instead of link styling for "Delete file"
- [Dev intent] Since there may be multiple files uploaded, delete file buttons should have specific labels. For example, "Delete document_one_representative.pdf" instead of just "Delete file"
- Either rewrite content so that "on behalf of" is not an isolated `p` tag between two fields (preferred) OR use `aria-describedby` so that it is programatically linked to the first field [Step 5 of 5: Review Application](https://preview.uxpin.com/3bf6496017f55041a94c2cfc8009c35dad5a79f2#/pages/137666520/simulate/sitemap?mode=i)
- ["Go back to VA.gov" should be styled as a link, not a button](https://preview.uxpin.com/3bf6496017f55041a94c2cfc8009c35dad5a79f2#/pages/137666521/simulate/sitemap?mode=i)

## Consider
- Style the `legend` to be more recognizable from standard `p` text [Step 4 of 5: Legal Representative](https://preview.uxpin.com/3bf6496017f55041a94c2cfc8009c35dad5a79f2#/pages/137666495/simulate/sitemap?mode=i)
- Place the the "What type of document..." hint accordion _above_ the upload button. Screen reader users will read the page by the order of elements in the DOM, so if the information within the hint accordion is important to interpreting what documents to upload it would be more convenient to place it before the upload button.
- Do not italicize information regarding file sizes. Long strings of italicized text can be difficult to read for some users with low vision or reading disabilities.
- Split the final review page into several pages. Given the length of content that must be read inbetween fields, the statements of truth should belong within their own step (e.g. Step 6: Sign application) with their own separate pages to reduce complexity of the page.

## Questions
- Has the scenario of multiple file uploads been considered?

**Comment by 1Copenut** [comment]
This is 💯 @joshkimux. VSP concurs with your feedback.

**Comment by noahgelman** [comment]
2nd approval from VSP. Thank you @joshkimux

**Comment by sshein** [comment]
Thanks Josh! So I'm a little confused regarding how much of the Upload element we should be changing, given that this has been recently documented in the design system by @CrystabelReiter (e.g. your suggestion regarding the delete file button copy, and your thoughts about italicized text) - Crystabel, can you sync up with Josh and Allison and make sure whatever is in the design system reflects suggestions by Josh & Allison?

That said, this feature will actually only allow one file to be uploaded once it's completed it turns out so we'll remove the "add another file" button. 

Everything else makes sense thanks although not sure we'll be able to split the review page, we'll probably keep that as-is for now

**Comment by sshein** [comment]
posted this request to platform support https://dsva.slack.com/archives/CBU0KDSB1/p1618937721211000
